core/vm/opcodes.go

```diff
@@ -200,8 +200,8 @@ const (
 
 // 0xb0 range - subroutine ops.
 const (
-	JUMPSUB   OpCode = 0xb3
-	BEGINSUB  OpCode = 0xb5
+	JUMPSUB   OpCode = 0xb2
+	BEGINSUB  OpCode = 0xb3
```

Comment:
this is still different from the spec :sweat_smile: 
> 0xb2 BEGINSUB
0xb3 JUMPSUB
0xb7 RETURNSUB

Reply:
Ah damnit :)

Reply:
Ok, changed it again, and updated the example code in the eip